devel/zmqpp/Portfile

```diff
@@ -17,7 +16,11 @@ long_description    This C++ binding for 0mq/zmq is a 'high-level' library \
                     It consists of a number of header and source files all residing in the zmq directory, \
                     these files are provided under the MPLv2 license (see LICENSE for details).
 
-depends_lib-append  port:zmq
+depends_lib-append  path:lib/libzmq.dylib:zmq
```

Comment:
shouldn't this change require a `revision` increase as you now allow other library dependencies?

Reply:
That does not change anything already installed (it merely allows to build against `zmq-devel`). So probably not.

Reply:
I admit that I'm not 100% sure about it, so let's leave it out..